### PR TITLE
Add myself to Team, and fix typos.

### PR DESCRIPTION
**Pull request by DiemenDesign**

### CODE REVIEW

File: 03-plugins.md [hunk 1]
```diff
@@ -55,9 +54,9 @@ Adding interaction to the Toolbar.
 $(document).ready(function() {
   $('#summernote').summernote({
     toolbar:[
-      // This is a Custom Button in a new Toolbar Area
+// This is a Custom Button in a new Toolbar Area
```
Comment (lqez) [inline 03-plugins.md]: Is removing blanks(indents) better for readability?

Comment (DiemenDesign) [reply 03-plugins.md]: I thought so, just to make things easier to read.

Comment (hackerwins) [reply 03-plugins.md]: I thought it is better to include indentation than removing for readability.

Comment (DiemenDesign) [reply 03-plugins.md]: I fixed the indentation so it's the same as other areas of the Website.

File: 03-plugins.md [hunk 2]
```diff
@@ -71,7 +70,7 @@ $(document).ready(function() {
   $('#summernote').summernote({
     popover: {
       image: [
-        // This is a Custom Button in a new Toolbar Area
+// This is a Custom Button in a new Toolbar Area
```
Comment (hackerwins) [inline 03-plugins.md]: I thought you missed this indentation.

Comment (DiemenDesign) [reply 03-plugins.md]: Yep, looks like I did, I'll check again, ooopppsss

**Pull request merged**